contracts/ManagedContract.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity 0.5.16;
+
+import "./Lockable.sol";
+
+contract ManagedContract is Lockable {
```

Comment:
This is ok but convoluted:
Contract is ManagedContract
ManagedContract is Lockable
Lockable js ContractRegistryAccessor

Are there contracts that are lockable and not not managed?

Reply:
There shouldn't be
(`StakingContractHandler` is accidentally)
Do you suggest merging them?

Reply:
Another option is not to have managed is lockable and have:
Contract is ManagedContract, Lockable
ManagedContract is ContractRegistryAccessor
Lockable js ContractRegistryAccessor